model/src/main/java/org/cloudfoundry/identity/uaa/provider/IdentityProvider.java

```diff
@@ -208,6 +232,8 @@ public int hashCode() {
         result = prime * result + ((name == null) ? 0 : name.hashCode());
         result = prime * result + ((originKey == null) ? 0 : originKey.hashCode());
         result = prime * result + ((type == null) ? 0 : type.hashCode());
+        result = prime * result + ((aliasId == null) ? 0 : aliasId.hashCode());
+        result = prime * result + ((aliasZid == null) ? 0 : aliasZid.hashCode());
```

Comment:
To be clarified: are these hash codes persisted anywhere and might therefore become a breaking change?

Reply:
Do we have an answer to this?

Reply:
hashCode is needed for serialization and compare (means to get the info about changes in an object)

Reply:
They are not persisted anywhere, correct?

Reply:
We also assumed that they are not persisted in any way. Therefore, I will mark this discussion as resolved.